mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -116,6 +116,12 @@
     <string name="recent_tabs_header">Jump back in</string>
     <!-- Button text for showing all the tabs in the tabs tray -->
     <string name="recent_tabs_show_all">Show all</string>
+    <!-- Content description for the button which navigates the user to show all recent tabs in the tabs tray. -->
+    <string name="recent_tabs_show_all_content_description">Show all recent tabs button</string>
```

Comment:
@eliserichards Should "button" appear at the end of this string? not entirely sure

Reply:
Yes, this is intentional 👍

Reply:
Thanks Elise! Can you help fix the other pending issues here as well perhaps? There are still quite a few scattered around this PR